src/domain/occurrence/__tests__/OccurrenceDetailsPage.test.tsx

```diff
@@ -0,0 +1,165 @@
+// import { axe, toHaveNoViolations } from 'jest-axe';
+import React from 'react';
+
+import {
+  EventDocument,
+  OccurrenceDocument,
+  PlaceDocument,
+  VenueDocument,
+} from '../../../generated/graphql';
+import {
+  renderWithRoute,
+  screen,
+  userEvent,
+  waitFor,
+} from '../../../utils/testUtils';
+import { ROUTES } from '../../app/routes/constants';
+import OccurrenceDetailsPage from '../OccurrenceDetailsPage';
+import {
+  eventResult,
+  occurrenceResult,
+  placeResult,
+  venueResult,
+} from './mocks';
+
+// expect.extend(toHaveNoViolations);
+
+const eventId = 'palvelutarjotin:afzunowba4';
+const occurrenceId = 'T2NjdXJyZW5jZU5vZGU6MTIz';
+const testPath = ROUTES.OCCURRENCE_DETAILS.replace(':id', eventId).replace(
+  ':occurrenceId',
+  occurrenceId
+);
+
+const mocks = [
+  {
+    request: {
+      query: EventDocument,
+      variables: {
+        id: eventId,
+        include: ['location'],
+      },
+    },
+    result: eventResult,
+  },
+  {
+    request: {
+      query: OccurrenceDocument,
+      variables: {
+        id: occurrenceId,
+      },
+    },
+    result: occurrenceResult,
+  },
+  {
+    request: {
+      query: PlaceDocument,
+      variables: {
+        id: 'tprek:15376',
+      },
+    },
+    result: placeResult,
+  },
+  {
+    request: {
+      query: VenueDocument,
+      variables: {
+        id: 'tprek:15376',
+      },
+    },
+    result: venueResult,
+  },
+];
+
+// uncomment this when enrolment table is accessible (labels for checkboxes)
```

Comment:
This is really minor. Maybe you could add "TODO:" at the beginning of the comment so it's easier to find in the future

Reply:
Yes you are right :)